### ISSUE TITLE
Could not compile :edeliver

### ISSUE THREAD

**Issue by bossyang** [boldpoker/edeliver#43]
I can't compile edeliver.
````
Could not compile :edeliver, no "mix.exs", "rebar.config" or "Makefile" (pass :compile as an option to customize compilation, set it to "false" to do nothing)
````
my mix.exs
````
  def application do
    [mod: {Pxblog, []},
     applications: [:phoenix, :phoenix_html, :cowboy, :logger, :gettext,
                    :phoenix_ecto, :postgrex, :comeonin, :earmark, :conform,
                    :conform_exrm, :edeliver]]
  end
  defp deps do
    [{:phoenix, "~> 1.1.4"},
     {:postgrex, ">= 0.0.0"},
     {:phoenix_ecto, "~> 2.0"},
     {:phoenix_html, "~> 2.4"},
     {:phoenix_live_reload, "~> 1.0", only: :dev},
     {:gettext, "~> 0.9"},
     {:cowboy, "~> 1.0"},
     {:comeonin, "~> 2.1"},
     {:ex_machina, "~> 0.6", only: :test},
     {:earmark, "~> 0.2.1"},
     {:exrm, "~> 1.0.0-rc7"},
     {:conform, "~> 1.0.0-rc8"},
     {:conform_exrm, "~> 0.2.0"},
     {:edeliver, "~> 1.1"}
   ]
  end

````

**Comment by mk** [comment]
Hi and thanks a lot for your report. It seems I messed up the release (I added `mix.exs` to hex when I released v1.1.1 and didn't push the changes. 🙀

Anyway, we've just released 1.1.3 which we've also tested this time to make sure it works. Hope you'll give edeliver another try.

**Comment by bossyang** [comment]
Thanks. release 1.1.3 works.
